java/client/src/org/openqa/selenium/By.java

```diff
@@ -44,6 +44,10 @@
  * </code>
  */
 public abstract class By {
+  static {
```

Comment:
the only driver that doesn't is HtmlUnitDriver and in that case, you could call in that constructor.